client/styles/welcome.scss

```diff
@@ -148,11 +146,19 @@ body {
   border-radius: 5px;
   width: 130px;
   height: 37px;
+
+  top: 225px;
+  padding: 8px, 12px, 8px, 12px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 123px;
```

Comment:
It breaks the button in the welcome page.
If you can, use a different styling for your page

Reply:
thats not myy componenet.. talk to shoval. i guess it merged from main when i pulled it:)

Reply:
weird, then it shouldnt appear here